### PR TITLE
feat: better paddings in modal section :lipstick:

### PR DESCRIPTION
**Pull request by gregorylegarec**
capture d ecran 2017-06-02 a 17 29 39

### CODE REVIEW

**Comment by GoOz** [comment]
Yeah don't step on each other on this. @y-lohse did you merge you commit on this on Photos yet?

File: stylus/ui-components/modals.styl [hunk 1]
```diff
@@ -52,17 +52,23 @@ $modals
...
+        .coz-modal-content
             padding 0 1.5em
 
         .coz-modal-section
+            padding 1.5em
+
+        .coz-modal-section
             border-top 1px solid grey-09
 
+        .coz-modal-content:first-of-type
```
Comment (GoOz) [inline stylus/ui-components/modals.styl]: `:first-of-type` is dangerous ground. This is not really maintenance friendly.
I don't know what you're reffering to but it seems to me that better class architecture would be a better option.
I get that the use of `:first-of-type` was here before but it gets more and more messy which just prove that maintenance will be difficult.

**Comment by y-lohse** [comment]
No nothing merged so far onmy end, but il will be pretty soon.

File: stylus/ui-components/modals.styl (hunk 1 above)
Comment (gregorylegarec) [inline stylus/ui-components/modals.styl]: What do you suggest ? Use some kind of `coz-modal-content--first` class ? It's a pain to implement and always complicate code.
Do you have some examples of maintenance issues with `:first-of-type` ?

Comment (GoOz) [inline stylus/ui-components/modals.styl]: I'll have a look, I must have a global look to know what would be best. Stay tuned.

**Comment by gregorylegarec** [comment]
Out of date and not needed anymore.

**Pull request closed without merging**